lib/debugger/ui.lua

```diff
@@ -160,6 +169,8 @@ local function ui(debugger, loop)
 						selected = debugger.debugSystem == system,
 						system = system,
 						icon = icon,
+						barWidth = duration / longestDuration,
```

Comment:
off hand, do we need to worry about anything if longestDuration is 0?

Reply:
No, because longestDuration will get set to a non-zero number if there's a system, and this code is in a loop over systems.

Reply:
too clever for our own good

Reply:
its not very clear there's a implicit dependency on that fact with the given control flow, but I do believe its unlikely to happen